<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUITransformGenerator</class>
 <widget class="QToolBox" name="GUITransformGenerator">
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="translate_widget">
   <attribute name="label">
    <string>Translation</string>
   </attribute>
   <layout class="QFormLayout" name="translate_layout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Δx</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="translate_x">
      <property name="minimum">
       <double>-99.989999999999995</double>
      </property>
      <property name="maximum">
       <double>99.989999999999995</double>
      </property>
      <property name="singleStep">
       <double>0.200000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Δy</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="translate_y">
      <property name="minimum">
       <double>-99.989999999999995</double>
      </property>
      <property name="maximum">
       <double>99.989999999999995</double>
      </property>
      <property name="singleStep">
       <double>0.200000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Δz</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="translate_z">
      <property name="minimum">
       <double>-99.989999999999995</double>
      </property>
      <property name="maximum">
       <double>99.989999999999995</double>
      </property>
      <property name="singleStep">
       <double>0.200000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="translate">
      <property name="text">
       <string>Generate Translation</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="scale_widget">
   <attribute name="label">
    <string>Scaling</string>
   </attribute>
   <layout class="QFormLayout" name="scale_layout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>αx</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="scale_x">
      <property name="minimum">
       <double>-99.989999999999995</double>
      </property>
      <property name="maximum">
       <double>99.989999999999995</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.200000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>αy</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="scale_y">
      <property name="minimum">
       <double>-99.989999999999995</double>
      </property>
      <property name="maximum">
       <double>99.989999999999995</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.200000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>αz</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="scale_z">
      <property name="minimum">
       <double>-99.989999999999995</double>
      </property>
      <property name="maximum">
       <double>99.989999999999995</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.200000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="scale">
      <property name="text">
       <string>Generate Scaling</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="rotate_widget">
   <attribute name="label">
    <string>Rotation</string>
   </attribute>
   <layout class="QFormLayout" name="rotate_layout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::ExpandingFieldsGrow</enum>
    </property>
    <item row="1" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>θx</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>θy</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>θz</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QPushButton" name="rotate">
      <property name="text">
       <string>Generate Rotation</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="rotate_z">
      <property name="minimum">
       <number>-360</number>
      </property>
      <property name="maximum">
       <number>360</number>
      </property>
      <property name="singleStep">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="rotate_y">
      <property name="minimum">
       <number>-360</number>
      </property>
      <property name="maximum">
       <number>360</number>
      </property>
      <property name="singleStep">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="rotate_x">
      <property name="minimum">
       <number>-360</number>
      </property>
      <property name="maximum">
       <number>360</number>
      </property>
      <property name="singleStep">
       <number>5</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
